ferveo-tdec/src/ciphertext.rs

```diff
@@ -30,7 +31,7 @@ pub struct Ciphertext<E: Pairing> {
     pub auth_tag: E::G2Affine,
 
     // V
-    #[serde(with = "serde_bytes")]
+    #[serde(with = "SliceLike::<Hex>")]
```

Comment:
Is this specifically serializing as hex instead of bytes? Backwards compatibility?

Reply:
It's serializing as hex in human-readable formats, and as bytes in binary formats. Same as in `umbral`. The binary format ABI won't change, unless `serde_bytes` was saving constant-sized byte arrays as `serialize_tuple()` in `serde`, which leads to inefficient serialization. I suppose I need to add vector tests here as well.

Reply:
Yeah. Some vector tests would be good for reinforcement.

Reply:
Added a big serialization vector test in `ferveo`, before the `serde-encoded-bytes` commit.